src/rules/selector-no-empty/README.md

```diff
@@ -0,0 +1,47 @@
+# selector-no-empty
+
+Disallow empty selectors.
+
+```css
+  a, , b {}
+/** ↑ 
+ * An empty selector */
+```
+
+Empty selectors (by themselves or within a selector list) invalidate a rule. This rule splits the selector list on `,`. Any part of zero length or containing only whitespace is deemed empty.
```

Comment:
@davidtheclark I think you should discuss this issue with @jeddy3, because it was his idea.